features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ThrottlingPolicies/BusinessPlans.js

```diff
@@ -0,0 +1,141 @@
+import TextField from 'material-ui/TextField';
+
+import API from '../../data/api'
+
+export default class BusinessPlans extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            policies: null,
+            selectedRowKeys: [],
+            open:false,
+            message: ''
+        };
+
+    }
+    componentDidMount() {
+        const api = new API();
+
+        const promised_policies = api.getSubscriptionLevelPolicies();
+        /* TODO: Handle catch case , auth errors and ect ~tmkb*/
+        promised_policies.then(
+            response => {
+               this.setState({policies: response.obj.list});
+            }
+        );
+    }
+    render() {
+        /*TODO implement search and pagination*/
+        const tiers = this.state.policies;
+        let data = [];
+        if(tiers) {
+          data = tiers;
+        }
+
+        return (
+            <div>
+                <AppBar position="static" >
+                    <Toolbar style={{minHeight:'30px'}}>
+                        <IconButton color="contrast" aria-label="Menu">
+                            <MenuIcon />
+                        </IconButton>
+                        <Link to={"/policies/business_plans/create/"}>
+                             <Button color="contrast">Add Plan</Button>
+                        </Link>
+                    </Toolbar>
+                </AppBar>
+                <Grid container>
+                    <Grid item xs={12}>
+                        <Paper>
+                            <Typography className="page-title" type="display1" gutterBottom>
+                               Business Plans
+                            </Typography>
+                            <Typography type="caption" gutterBottom align="left" className="page-title-help">
+                            Discription goes here.
+                            </Typography>
+
+                            <Divider />
+                            <div className="page-content">
+                              <TextField
+                                label="Search"
+                                margin="normal"
+                              />
+                            </div>
+                            <Divider />
+                        </Paper>
+                    </Grid>
+                    <Grid item xs={12} className="page-content">
+                          <Paper>
+                            <Table>
+                              <TableHead>
+                                <TableRow>
+                                  <TableCell>Name</TableCell>
+                                  <TableCell>Quota Policy</TableCell>
+                                  <TableCell>Quota</TableCell>
+                                  <TableCell>Unit Time</TableCell>
+                                  <TableCell>Rate Limit</TableCell>
+                                  <TableCell>Time Unit</TableCell>
```

Comment:
what is the difference between unti time? and time unit?

Reply:
time unit column displays the units used to measure the rate limit. Unit time column is used to show the unite which is used for the quota  (ex: 1000 request for 1 min. 1min is the unit time). This is same used in AM 2.1 as well

Reply:
If that is so, can we not have the two columns together?